ratbagd/ratbagd-device.c

```diff
@@ -222,7 +246,8 @@ ratbag_device_get_model(sd_bus *bus,
 
 const sd_bus_vtable ratbagd_device_vtable[] = {
 	SD_BUS_VTABLE_START(0),
-	SD_BUS_PROPERTY("Model", "s", ratbag_device_get_model, 0, SD_BUS_VTABLE_PROPERTY_CONST),
+	SD_BUS_PROPERTY("Model", "s", ratbagd_device_get_model, 0, SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
see above

Reply:
@K1ngjulien if you look at the commit history you did a minor mistake - in your second commit, you renamed it from `Model` to `DeviceType` and added `Model` again in the third commit, please change that.

Reply:
yeah, sorry

Reply:
rebase did it again.... i'll try and fix it again.....